packages/pn-personafisica-webapp/src/App.tsx

```diff
@@ -211,10 +212,16 @@ const App = () => {
       ),
   });
 
+  const handleLogout = () => {
+    void dispatch(logout());
+
+    goToLoginPortal(window.location.origin, true);
```

Comment:
@LorenzoCocciarelli come per l'altro commento. Inoltre, se handlelogout fa sempre le stesse cose in ogni punto in cui è stato implementato, creare un metodo di utility da usare quando necessario